### PR TITLE
new: pings

### PR DESCRIPTION
**Pull request by sibicramesh**

### CODE REVIEW

File: specs/pingprobe.spec [hunk 1]
```diff
@@ -1,11 +1,17 @@
 # Model
 model:
-  rest_name: pingreport
-  resource_name: pingreports
-  entity_name: PingReport
-  package: zack
+  rest_name: pingprobe
+  resource_name: pingprobe
```
Comment (primalmotion) [inline specs/pingprobe.spec:4]: pingprobes

File: specs/pingrequest.spec [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# Model
+model:
+  rest_name: pingrequest
+  resource_name: pingrequest
```
Comment (primalmotion) [inline specs/pingrequest.spec:4]: pingrequests

File: specs/pingrequest.spec [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+  entity_name: PingRequest
+  package: guy
+  group: core/enforcer
```
Comment (primalmotion) [inline specs/pingrequest.spec:7]: Please align the group for all ping related objects

File: specs/pingrequest.spec [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+  description: Post a new pu pingrequest.
+
+# Attributes
+attributes:
+  v1:
+  - name: iterations
+    description: Number of probes that will be triggered.
+    type: integer
+    exposed: true
+    stored: true
```
Comment (primalmotion) [inline specs/pingrequest.spec:17]: I guess you missed the pingID , autogenerated, readonly that backend will reply with

File: specs/root.spec [hunk 5]
```diff
@@ -615,9 +615,14 @@ relations:
   create:
     description: Creates a new PCC provider.
 
-- rest_name: pingreport
+- rest_name: pingprobe
```
Comment (primalmotion) [inline specs/root.spec:618]: Missing pingreports and pingresults

File: specs/pingrequest.spec (hunk 3 above)
Comment (sibicramesh) [reply specs/pingrequest.spec]: Not sure what the new service will be. I just aligned the group to `core` and package `guy`.

**Comment by ericrpowers** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "pings",
    "component": "backend",
    "pr-id": "759",
    "commit-sha": "6c9ddd81516437f9fe00de0c5a70388a97cd6912",
    "pipeline": "master"
  },
  {
    "project": "pings",
    "component": "gaia",
    "pr-id": "500",
    "commit-sha": "8adea1b67e1c24ed8fa1c27cd223418e47206323",
    "pipeline": "master"
  }
]
```

**Comment by ericrpowers** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "pings",
    "component": "backend",
    "pr-id": "759",
    "commit-sha": "6c9ddd81516437f9fe00de0c5a70388a97cd6912",
    "pipeline": "master"
  },
  {
    "project": "pings",
    "component": "gaia",
    "pr-id": "500",
    "commit-sha": "8adea1b67e1c24ed8fa1c27cd223418e47206323",
    "pipeline": "master"
  }
]
```

**Comment by ericrpowers** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "pings",
    "component": "backend",
    "pr-id": "759",
    "commit-sha": "a824cecdea065fb14eead2729b904e4b11ff529b",
    "pipeline": "master"
  },
  {
    "project": "pings",
    "component": "gaia",
    "pr-id": "500",
    "commit-sha": "8adea1b67e1c24ed8fa1c27cd223418e47206323",
    "pipeline": "master"
  }
]
```

**Comment by sibicramesh** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "pings",
    "component": "backend",
    "pr-id": "759",
    "commit-sha": "37426ce739f4f6ee18225c3c0f09ab3c3532830e",
    "pipeline": "master"
  },
  {
    "project": "pings",
    "component": "gaia",
    "pr-id": "500",
    "commit-sha": "7f99e13001bcfc29fb52a452c650ed1b34dc33b4",
    "pipeline": "master"
  },
  {
    "project": "pings",
    "component": "apotests",
    "pr-id": "2622",
    "commit-sha": "ad17589583f7205d887eb73aaaf85f31f9eda4ac",
    "pipeline": "master"
  },
  {
    "project": "pings",
    "component": "enforcerd",
    "pr-id": "1737",
    "commit-sha": "920d00c33de2cd4a465cda3a45a27dd3cb2d201c",
    "pipeline": "master"
  }
]
```

**Comment by sibicramesh** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "pings",
    "component": "backend",
    "pr-id": "759",
    "commit-sha": "37426ce739f4f6ee18225c3c0f09ab3c3532830e",
    "pipeline": "master"
  },
  {
    "project": "pings",
    "component": "gaia",
    "pr-id": "500",
    "commit-sha": "7f99e13001bcfc29fb52a452c650ed1b34dc33b4",
    "pipeline": "master"
  },
  {
    "project": "pings",
    "component": "apotests",
    "pr-id": "2622",
    "commit-sha": "ad17589583f7205d887eb73aaaf85f31f9eda4ac",
    "pipeline": "master"
  },
  {
    "project": "pings",
    "component": "enforcerd",
    "pr-id": "1737",
    "commit-sha": "a53549ed63b716e4a9542a431d92b4ea2bfad109",
    "pipeline": "master"
  }
]
```

**Comment by sibicramesh** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "pings",
    "component": "gaia",
    "pr-id": "500",
    "commit-sha": "7f99e13001bcfc29fb52a452c650ed1b34dc33b4",
    "pipeline": "master"
  },
  {
    "project": "pings",
    "component": "apotests",
    "pr-id": "2622",
    "commit-sha": "ad17589583f7205d887eb73aaaf85f31f9eda4ac",
    "pipeline": "master"
  },
  {
    "project": "pings",
    "component": "enforcerd",
    "pr-id": "1737",
    "commit-sha": "81b651f903dff08628ac8ab3d77686744005be54",
    "pipeline": "master"
  },
  {
    "project": "pings",
    "component": "backend",
    "pr-id": "759",
    "commit-sha": "37426ce739f4f6ee18225c3c0f09ab3c3532830e",
    "pipeline": "master"
  }
]
```

**Comment by sibicramesh** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "pings",
    "component": "backend",
    "pr-id": "759",
    "commit-sha": "534806456c493be6fa6e559e0e269ba96b93db84",
    "pipeline": "master"
  },
  {
    "project": "pings",
    "component": "gaia",
    "pr-id": "500",
    "commit-sha": "a5558ec2ab9317ec2568a9aee2c7c8af7bac87ee",
    "pipeline": "master"
  },
  {
    "project": "pings",
    "component": "enforcerd",
    "pr-id": "1737",
    "commit-sha": "0cd6e91cc091cfef4a2733602e0cc2706e53c62f",
    "pipeline": "master"
  }
]
```

**Comment by sibicramesh** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "pings",
    "component": "backend",
    "pr-id": "759",
    "commit-sha": "534806456c493be6fa6e559e0e269ba96b93db84",
    "pipeline": "master"
  },
  {
    "project": "pings",
    "component": "gaia",
    "pr-id": "500",
    "commit-sha": "a5558ec2ab9317ec2568a9aee2c7c8af7bac87ee",
    "pipeline": "master"
  },
  {
    "project": "pings",
    "component": "enforcerd",
    "pr-id": "1737",
    "commit-sha": "0cd6e91cc091cfef4a2733602e0cc2706e53c62f",
    "pipeline": "master"
  }
]
```

**Comment by sibicramesh** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "pings",
    "component": "backend",
    "pr-id": "759",
    "commit-sha": "534806456c493be6fa6e559e0e269ba96b93db84",
    "pipeline": "master"
  },
  {
    "project": "pings",
    "component": "gaia",
    "pr-id": "500",
    "commit-sha": "a5558ec2ab9317ec2568a9aee2c7c8af7bac87ee",
    "pipeline": "master"
  },
  {
    "project": "pings",
    "component": "enforcerd",
    "pr-id": "1737",
    "commit-sha": "0cd6e91cc091cfef4a2733602e0cc2706e53c62f",
    "pipeline": "master"
  }
]
```

**Comment by sibicramesh** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "pings",
    "component": "backend",
    "pr-id": "759",
    "commit-sha": "534806456c493be6fa6e559e0e269ba96b93db84",
    "pipeline": "master"
  },
  {
    "project": "pings",
    "component": "gaia",
    "pr-id": "500",
    "commit-sha": "a5558ec2ab9317ec2568a9aee2c7c8af7bac87ee",
    "pipeline": "master"
  },
  {
    "project": "pings",
    "component": "enforcerd",
    "pr-id": "1737",
    "commit-sha": "0cd6e91cc091cfef4a2733602e0cc2706e53c62f",
    "pipeline": "master"
  }
]
```

**Pull request merged**